### PR TITLE
[script.module.pyjwt] 1.6.4

### PR DESCRIPTION
**Pull request by lottaboost**
pyjwt library adapted for kodi.  Allows encoding and decoding of jwt tokens

### CODE REVIEW

File: script.module.pyjwt/addon.xml [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+<addon id="script.module.pyjwt"
+       name="pyjwt"
+       provider-name="jpadilla (hello@jpadilla.com)" 
+       version="1.6.4">
+  <requires>
+    <import addon="xbmc.python" version="2.24.0" />
+  </requires>
+  <extension library="lib" point="xbmc.python.module" />
+  <extension point="xbmc.addon.metadata">
+        <description lang="en">pyjwt module</description>
```
Comment (Rechi) [inline script.module.pyjwt/addon.xml]: `lang="en_GB"`

File: script.module.pyjwt/addon.xml [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+        <summary lang="en">pyjwt module</summary>
```
Comment (Rechi) [inline script.module.pyjwt/addon.xml]: `lang="en_GB"`

File: script.module.pyjwt/addon.xml [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+        <license>MIT</license>
+        <platform>all</platform>
+        <email></email>
```
Comment (Rechi) [inline script.module.pyjwt/addon.xml]: remove empty tag

File: script.module.pyjwt/addon.xml [hunk 4]
```diff
@@ -0,0 +1,21 @@
...
+        <website>https://pyjwt.readthedocs.io</website>
+        <forum></forum>
```
Comment (Rechi) [inline script.module.pyjwt/addon.xml]: remove empty tag

File: script.module.pyjwt/addon.xml [hunk 5]
```diff
@@ -0,0 +1,21 @@
...
+        <source>https://github.com/jpadilla/pyjwt</source>
+        <language></language>
```
Comment (Rechi) [inline script.module.pyjwt/addon.xml]: remove empty tag

**Comment by lottaboost** [comment]
Ok, thanks will do

**Comment by Rechi** [comment]
Please update this PR, it is annoying that you open a new PR for every little change.

**Comment by Rechi** [comment]
From our pull request template
> Each add-on submission should be a single commit with using the following style: [plugin.video.foo] v1.0.0

So now please squash the two commits and adjust the commit message to mention the correct version.

**Comment by lottaboost** [comment]
I've never figured out how to squash commits properly, that's why I've done the annoying multiple PR requests, any suggestions?

**Comment by Rechi** [comment]
To fix your pull request execute the following commands (origin is your repo, upstream is the xbmc repo)

```
git fetch upstream
git checkout jarvis
git reset --hard upstream/jarvis
git checkout origin/jarvis-- script.module.pyjwt/
git commit -m '[script.module.pyjwt] 1.6.4'
git push -f origin jarvis
```

If you want to add changes to or change the message of an already existing commit you can do that with `git commit --amend`. Then just force push to your branch. (This doesn't work here any more as you have already create a second commit.)

**Comment by lottaboost** [comment]
Squashed the commits.  You have no idea how long I've been trying to properly squash PR commits.  I really appreciate it, thank you.
Brian

**Comment by Rechi** [comment]
Why have you changed the PR title? The version of this add-on is 1.6.4.
Also you still missed to fix the version in the git commit message. How could that happen, as I mentioned the right one in my instructions?

Now you have the chance to try the second method, `git commit --amend` and adjsust the commit message. After that force push again to your PR branch (`git push -f origin jarvis`).

**Comment by lottaboost** [comment]
Sorry for the trouble, I'm very new to git.  As far as the version, since this is the first script.module.pyjwt I put 1.0.0.  From now on, if I ever port a library over, I will use the version of the library itself.
Thanks Again, sorry for being a PITA.  The git info you provided was extremely helpfull.

**Pull request merged**